src/tests/functional/plugin/gpu/shared_tests_instances/multi/gpu_remote_blob_tests.cpp

```diff
@@ -50,29 +63,86 @@ TEST_P(MultiDevice_Test, cannotInferRemoteBlobIfNotInitializedForDevice) {
+    InferenceEngine::CNNNetwork net(fn_ptr);
+    auto ie = PluginCache::get().ie();
+    // load a network to the GPU to make sure we have a remote context
+    auto exec_net = ie->LoadNetwork(net, GPU);
+    auto ctx = exec_net.GetContext();
+
+    const InferenceEngine::ConstInputsDataMap inputInfo = exec_net.GetInputsInfo();
+    auto& first_input = inputInfo.begin()->second;
+    auto rblob = InferenceEngine::make_shared_blob(first_input->getTensorDesc(), ctx);
+    rblob->allocate();
+
+    std::map<std::string, std::string> configs;
+    for (auto&& value : _properties) {
+        configs.emplace(value.first, value.second.as<std::string>());
+    }
+
+    InferenceEngine::ExecutableNetwork exec_net_multi;
+    try {
+        exec_net_multi = ie->LoadNetwork(net, device_names, configs);
+    } catch(...) {
+        // device is unavailable (e.g. for the "second GPU" test) or other (e.g. env) issues not related to the test
+        return;
+    }
+
+    unsigned int optimalNum = 0;
+    try {
+        optimalNum = exec_net_multi.GetMetric(METRIC_KEY(OPTIMAL_NUMBER_OF_INFER_REQUESTS)).as<unsigned int>();
+    } catch (...) {
+        std::cout << "ExecutableNetwork getMetric failed" << std::endl;
+        return;
+    }
+
+    // test binder mode to throw exception when oversubsciption of infer requests
+    InferenceEngine::InferRequest req;
+    for (int i = 0; i < optimalNum; i++) {
+        req = exec_net_multi.CreateInferRequest();
+    }
+    ASSERT_ANY_THROW(req = exec_net_multi.CreateInferRequest());
+}
+
 auto device_names_and_support_for_remote_blobs2 = []() {
-    return std::vector<DevicesNames>{
+    return std::vector<DevicesNamseAndProperties>{
 #ifdef ENABLE_INTEL_CPU
-        {CPU},  // stand-alone CPU via MULTI (no GPU), no OCL context
+        //{{CPU}, {}},  // stand-alone CPU via MULTI (no GPU), no OCL context
+        {{CPU}, {ov::intel_auto::device_bind_buffer(true)}},  // stand-alone CPU via MULTI (no GPU), no OCL context
 #endif
-        {"GPU.1"},  // another GPU (the test will test its presence), different OCL contexts
+        {{"GPU.1"}, {}},  // another GPU (the test will test its presence), different OCL contexts
+        {{"GPU.1"}, {ov::intel_auto::device_bind_buffer(true)}},  // another GPU (the test will test its presence), different OCL contexts
     };
 };
 
-INSTANTIATE_TEST_SUITE_P(smoke_RemoteBlobMultiInitializedWithoutGPU,
+INSTANTIATE_TEST_SUITE_P(smoke_Multi_RemoteBlobInitializedWithoutGPU,
                          MultiDevice_Test,
                          ::testing::ValuesIn(device_names_and_support_for_remote_blobs2()),
                          MultiDevice_Test::getTestCaseName);
 
+auto device_names_and_support_for_remote_blobs3 = []() {
+    return std::vector<DevicesNamseAndProperties>{
+#ifdef ENABLE_INTEL_CPU
+        {{CPU}, {ov::intel_auto::device_bind_buffer(true)}},  // stand-alone CPU via MULTI (no GPU), no OCL context
+#endif
```

Comment:
if this is only cpu only, move to cpu func tests, or use gpu only, or just delete this test

Reply:
Updated, replace CPU with GPU.